ksql-functional-tests/src/test/resources/query-validation-tests/explode.json

```
@@ -0,0 +1,68 @@
+{
+  "comments": [
+    "Tests for the EXPLODE table function"
+  ],
+  "tests": [
+    {
+      "name": "explode array with values",
+      "statements": [
+        "CREATE STREAM TEST (MY_ARR ARRAY<BIGINT>) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT EXPLODE(MY_ARR) VAL FROM TEST;"
```

Might be worth adding other columns / literals in the projection to test/show how they work e.g.

```suggestion
        "CREATE STREAM TEST (ID INT, MY_ARR ARRAY<BIGINT>) WITH (kafka_topic='test_topic', value_format='JSON');",
        "CREATE STREAM OUTPUT AS SELECT ID, "literal" AS LITERAL, EXPLODE(MY_ARR) VAL FROM TEST;"
```

---

The more general tests for table functions do this already and are in table_functions.json. The tests in explode.json are specific to the functionality of explode